### PR TITLE
Feature/120 Knowledge Base Code Refactoring

### PR DESCRIPTION
**Pull request by mbugert**

### CODE REVIEW

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/IriConstants.java [hunk 1]
```diff
@@ -21,20 +21,36 @@
...
 public class IriConstants
 {
 
+    public static final String INCEPTION_SCHEMA_NAMESPACE = "http://www.ukp.informatik.tu-darmstadt.de/inception/schema-1.0#";
+
+    public static final String INCEPTION_NAMESPACE = "http://www.ukp.informatik.tu-darmstadt.de/inception/1.0#";
+
     public static final String WIKIDATA_NAMESPACE = "https://www.wikidata.org/wiki/";
 
+    public static final String[] IMPLICIT_NAMESPACES = { RDF.NAMESPACE, RDFS.NAMESPACE,
```
Comment (Rentier) [inline inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/IriConstants.java]: To make it simple, I would either use Set or array for a collection of IRI, but not both next to each other.

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBConcept.java [hunk 2]
```diff
@@ -53,8 +53,8 @@
     /* Commented out until the functionality which uses them is actually implemented
     static {
         ValueFactory factory = SimpleValueFactory.getInstance();
-        CLOSED = factory.createIRI(KnowledgeBaseService.INCEPTION_SCHEMA_NAMESPACE, "closed");
-        ABSTRACT = factory.createIRI(KnowledgeBaseService.INCEPTION_SCHEMA_NAMESPACE, "abstract");
+        CLOSED = factory.createIRI(IriConstants.INCEPTION_SCHEMA_NAMESPACE, "closed");
+        ABSTRACT = factory.createIRI(IriConstants.INCEPTION_SCHEMA_NAMESPACE, "abstract");
```
Comment (Rentier) [inline inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBConcept.java]: Do we still use these?

File: inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/Validators.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                validatable.error(new ValidationError(this));
+            }
+        }
+    };
+    public static final UrlValidator URL_VALIDATOR = new UrlValidator(
+    new String[] { "http", "https" });
+
+    public Validators()
+    {
+        // TODO Auto-generated constructor stub
```
Comment (Rentier) [inline inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/Validators.java]: Fix me pls.

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBConcept.java (hunk 2 above)
Comment (mbugert) [reply inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBConcept.java]: Like the comments say: not right now. I'd keep it the way it is (commented out).

**Comment by mbugert** [comment]
Jenkins fails during `maven-javadoc-plugin:3.0.0:aggregate`. @reckart could you look into it some time?

**Comment by reckart** [comment]
```
[ERROR] /home/svc_jenkins/workspace/INCEpTION (GitHub) (PR builder)/inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/ConceptInstancePanel.java:80: error: @param name not found
[ERROR] * @param target
[ERROR] /home/svc_jenkins/workspace/INCEpTION (GitHub) (PR builder)/inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/KnowledgeBasePanel.java:136: error: @param name not found
[ERROR] * @param target
```

@mbugert could you have a look at the conflicting files please?

**Pull request merged**